harvard-lil/website-static
# Update typography

## What this does 
Swaps in Neue Haas Grotesk for headings and body text throughout the LIL website. To avoid false italics and false bold being applied by browsers, this update additionally adjusts the font-weight for text site-wide, and uploads an italic style of  Neue Haas Grotesk. 

I noticed that the website was previously applying a bold weight site-wide and then removing it certain case-by-case instances. This update reverts all text to a roman/body weight, and applies a bold weight as needed. 

## Screenshots

Homepage hero 
![Screenshot 2024-02-29 at 4 04 43 PM](https://github.com/harvard-lil/website-static/assets/4039311/0a0de0ab-eb18-4270-bc44-6b910e0d35a9)
Note: the line width of the introduction text, shown at large viewport sizes like what is pictured above, will be adjusted in a future update.

Homepage about section
![Screenshot 2024-02-29 at 4 06 45 PM](https://github.com/harvard-lil/website-static/assets/4039311/d224abd3-a9a5-47e8-ae6f-1324a62844fe)

Individual projects, from the projects page
![Screenshot 2024-02-29 at 4 06 54 PM](https://github.com/harvard-lil/website-static/assets/4039311/4c90793b-6f97-45ac-9876-92cfec875bfb)

Headshots, from the about page
![Screenshot 2024-02-29 at 4 07 15 PM](https://github.com/harvard-lil/website-static/assets/4039311/45ab1b07-7ffc-4897-ada0-25d7a669784a)

Blog post
![Screenshot 2024-03-04 at 9 40 47 AM](https://github.com/harvard-lil/website-static/assets/4039311/3ac6f795-408c-4315-963c-e9e0c8b500fd)

Italic body text in blog post
![Screenshot 2024-03-04 at 9 41 10 AM](https://github.com/harvard-lil/website-static/assets/4039311/9f3c7a4f-e9dc-4533-860e-bc93da65412a)

## How To Test
I've discovered that the least-error prone way to check out a branch locally is to not follow the instructions github provides on each PR. It feels much more consistent to:

- Add a new remote for my fork of website-static, named meaningfully (for example, named tinykite)
- Run `git pull tinykite` to make sure you have the latest remote branches
- Run `git checkout update-typography` to checkout this branch
